apps/doc/docs/intro.md

```diff
@@ -2,58 +2,132 @@
 sidebar_position: 1
 ---
 
-# Datatlas
+# 🌍 Datatlas
 
-Let's discover **Docusaurus in less than 5 minutes**.
+## What is DatAtlas ?
 
-## Getting Started
+DatAtlas is a project thats aims at creating the Wordpress for maps ! Thanks to this beautiful tool you will be able to create custom web maps by selecting your data, choosing your features, adding your style. Once your map is created you can embed it in any website you want. You want to create beautiful maps with 3D data ? With lot of layers ? With specific features ? You are at the right place !
 
-Get started by **creating a new site**.
+## 🐤 Getting started
 
-Or **try Docusaurus immediately** with **[docusaurus.new](https://docusaurus.new)**.
+Datatlas is still under heavy development.
+You can deploy it locally for development purpose.
+The production version will be available soon.
 
-dfdfd
+### ⌨️ Development
 
-```mermaid
-  graph TD;
-      A-->B;
-      A-->C;
-      B-->D;
-      C-->D;
+#### Prerequisites
+
+- [Node.js](https://nodejs.org/en/download/)
+- [npm](https://www.npmjs.com/get-npm)
+- [Docker](https://docs.docker.com/get-docker/)
+
+#### ⚙️ Installation
+
+```bash
+git clone https://github.com/datatlas-erasme/datatlas
+cd datatlas
+git checkout dev
+npm install
 ```
 
-eee
+#### 🔥 Run
 
-### What you'll need
+Run the postgres database and pgadmin with docker
 
-- [Node.js](https://nodejs.org/en/download/) version 16.14 or above:
-  - When installing Node.js, you are recommended to check all checkboxes related to dependencies.
+```bash
+docker-compose up -f dev.docker-compose.yml
 
-## Generate a new site
+```
 
-Generate a new Docusaurus site using the **classic template**.
+In another shell, run the backend
 
-The classic template will automatically be added to your project after you run the command:
+```bash
+npx nx serve backend
+```
+
+In another shell, run the frontend
 
 ```bash
-npm init docusaurus@latest my-website classic
+npx nx serve frontend
 ```
 
-You can type this command into Command Prompt, Powershell, Terminal, or any other integrated terminal of your code editor.
+You can optionnaly run the documentation
 
-The command also installs all necessary dependencies you need to run Docusaurus.
+```bash
+npx nx serve doc
+```
 
-## Start your site
+You can now access :
 
-Run the development server:
+- the frontend at http://localhost:3000
+- the backend at http://localhost:3333
+- pgadmin at http://localhost:5431
+- the documentation at http://localhost:3001
+
+#### 🔎 Test and lint
+
+Jest, Cypress and husky are used for testing and 🌸 linting.
+
+** 🐺 Husky**
+
+Please install husky on your local machine to run the pre-commit hook
+
+```bash
+npm install -g husky
+```
+
+Use 'nx format:write' before commiting to format the code with prettier
+
+**Frontend**
 
 ```bash
-cd my-website
-npm run start
+npx nx run frontend:test
```

Comment:
Shall we mention the command to run them all instead ?
```
npx nx run-many --target=test
```
We already have many apps and if you mention 2 of them you should probably mention all of them.... instead this command allow to doc to scale to many more apps !